packages/azure-openapi-validator/autorest/package.json

```diff
@@ -64,7 +64,7 @@
     "@stoplight/types": "^13.5.0",
     "dependency-graph": "^0.11.0",
     "@microsoft.azure/openapi-validator-core": "~1.0.0",
-    "@microsoft.azure/openapi-validator-rulesets": "^1.1.1",
+    "@microsoft.azure/openapi-validator-rulesets": "^1.1.2",
```

Comment:
You mean there isnt any necessity for ^ here right?
Also the latest version is 1.2.0 and now will be 1.2.1 and should this still be 1.1.*?

Reply:
Observe the version is `^1.1.1`. Which means it will accept anything of form `1.X.Y` where `X >= 1` and `Y >= 1`. So given you are changing ruleset from `1.2.0` to `1.2.1`, it will pick it up. Also observe you are bumping it here from `1.1.1` to `1.1.2` but you changed the ruleset from `1.2.0` to `1.2.1`.

Reply:
that makes sense. thanks so much!